jsapp/scss/components/_kobo.table.scss

```diff
@@ -21,217 +26,329 @@
+              padding: 0 5px 0 35px;
+            }
+
+            .popover-menu__content {
+              left: 5px;
+            }
+          }
         }
       }
     }
 
-    .rt-thead {
-      &.-filters input {
-        padding: 2px 4px;
-      }
+    // the second header row
+    &.-filters {
+      overflow-x: visible;
+      // we want both header rows to look like one merged
+      border-bottom: $data-table-border;
+      border-bottom-width: 2px;
 
-      &.-filters select {
-        margin-top: 2px;
-        margin-right: 2px;
+      .rt-th,
+      .rt-td {
+        border-right: $data-table-border;
       }
 
-      .rt-th:hover,
-      .rt-td:hover {
-        background: $kobo-teal-light;
-      }
+      .table-filter-input {
+        width: 100%;
+        border: 0;
+        background: $kobo-white;
+        border-radius: 3px;
+        height: 28px;
+        min-height: 28px;
+        vertical-align: top;
+        display: inline-block;
+        font-size: 12px;
 
-      .rt-th.-sort-desc,
-      .rt-td.-sort-desc {
-        box-shadow: inset 0 -3px 0 0 rgba($kobo-teal, 0.8);
+        &::placeholder {
+          color: $kobo-gray-55;
+        }
       }
 
-      .rt-th.-sort-asc,
-      .rt-td.-sort-asc {
-        box-shadow: inset 0 3px 0 0 rgba($kobo-teal, 0.8);
+      select {
+        margin-top: 2px;
+        margin-right: 2px;
       }
     }
 
-    .rt-tr-group {
-      max-height: 36px;
+    .rt-tr {
+      align-items: center;
     }
 
-    .rt-tr {
-      align-items: stretch;
+    .rt-th:hover,
+    .rt-td:hover {
+      background: transparent;
     }
 
-    .rt-th, .rt-td {
-      padding: 2px;
-      padding-left: 6px;
-      border-right: 1px solid rgba($kobo-gray-24, 0.03);
-      line-height: 30px;
+    .rt-th.-sort-desc,
+    .rt-td.-sort-desc,
+    .rt-th.-sort-asc,
+    .rt-td.-sort-asc {
+      box-shadow: inset 0 3px 0 0 $kobo-teal;
     }
 
-    .rt-td.rt-status {
-      line-height: inherit;
+    .rt-resizable-header-content {
       overflow: visible;
+      text-overflow: inherit;
 
-      .kobo-select__control {
-        background: transparent;
-        height: 30px;
-        min-height: 30px;
-        border: none;
-        border-radius: 0px;
-        box-shadow: none;
-        cursor: pointer;
+      .column-header-title,
+      .column-header-hxl-tags {
+        display: block;
+        width: 100%;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
 
-        &:hover {
-          box-shadow: none;
-        }
+      .column-header-hxl-tags {
+        color: $kobo-gray-40;
       }
     }
+  }
 
-    .rt-link {
-      overflow: visible;
+  .rt-tr {
+    align-items: stretch;
+    justify-content: stretch;
+  }
 
-      .table-link {
-        cursor: pointer;
-        display: inline-block;
-        min-height: 28px;
+  .rt-tr-group {
+    max-height: $data-table-row-height;
+  }
 
-        &:hover {
-          color: $kobo-blue;
-        }
+  .rt-th, .rt-td {
```

Comment:
Not on this line but related to `rt-td`, a slightly obscure bug:
At around 830px wide one of the translation cells has an added padding
![image](https://user-images.githubusercontent.com/16762675/121826516-04043280-cc86-11eb-8266-3e593a900fd6.png)

and around 430px it gets added to both
![image](https://user-images.githubusercontent.com/16762675/121826549-3877ee80-cc86-11eb-8f21-b085807d1e92.png)

This is obviously not a big deal, but is a change from production that seems odd

Reply:
This is actually due to first column text wrapping into new line. In first screenshot only second row is wrapping, in second both of them are.